extras-requirements.txt

```diff
@@ -33,4 +33,7 @@ kafka-python~=2.0
 avro~=1.11
 redis~=4.3
 graphviz~=0.20.0
-databricks-sdk~=0.3.0
\ No newline at end of file
+databricks-sdk~=0.3.0
+# sqlalchemy and pymysql versions should be aligned with mlrun-api due to common use with the model monitoring database
+sqlalchemy~=1.4
+pymysql~=1.0
```

Comment:
pymysql is only needed in mlrun/mlrun. can be moved to dockerfiles/mlrun/requirements.txt